src/repositories/media.repository.ts

```diff
@@ -127,6 +128,38 @@ export class MediaRepository extends Repository {
     });
   }
 
+  public async checkOffensiveComment(
```

Comment:
What is this?) If it's not so hard for you, could you please explain unclear endpoints with tsdos? Just text description will be enough